docs/src/content/docs/hi/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,289 @@
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Tailwind के साथ Starlight को स्टाइल करें
+
+Starlight अपने UI में आपके [Tailwind थीम कॉन्फिगरेशन](https://tailwindcss.com/docs/theme) से मूल्यों का उपयोग करेगा।
+
+यदि सेट किया गया है, तो निम्नलिखित विकल्प Starlight की डिफ़ॉल्ट शैलियों क अवहेलन कर देंगे:
+
+- `colors.accent` — लिंक और वर्तमान वस्तु हाइलाइटिंग के लिए उपयोग किया जाता है
+- `colors.gray` — पृष्ठभूमि रंगों और सीमाओं के लिए उपयोग किया जाता है
+- `fontFamily.sans` — UI और सामग्री पाठ के लिए उपयोग किया जाता है
+- `fontFamily.mono` — कोड उदाहरणों के लिए उपयोग किया जाता है
+
+```js {12,14,18,20}
+// tailwind.config.mjs
+import starlightPlugin from '@astrojs/starlight-tailwind';
+import colors from 'tailwindcss/colors';
+
+/** @type {import('tailwindcss').Config} */
+export default {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// आपका पसंदीदा उच्चारण रंग. Indigo Starlight के डिफॉल्ट्स के सबसे करीब है।
+				accent: colors.indigo,
+				// आपका पसंदीदा ग्रे स्केल। Zinc Starlight के डिफॉल्ट्स के सबसे करीब है।
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// आपका पसंदीदा टेक्स्ट फॉण्ट। Starlight डिफ़ॉल्ट रूप से system फ़ॉन्ट स्टैक का उपयोग करता है।
+				sans: ['"Atkinson Hyperlegible"'],
+				// आपका पसंदीदा कोड फॉण्ट। Starlight डिफ़ॉल्ट रूप से system monospace फॉण्ट का उपयोग करता है।
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Theming
+
+Starlight की रंग थीम को उसके डिफ़ॉल्ट कस्टम गुणों का अवहेलन करके नियंत्रित किया जा सकता है।
+इन चरों का उपयोग पूरे UI में टेक्स्ट और पृष्ठभूमि रंगों के लिए उपयोग किए जाने वाले ग्रे शेड्स की एक श्रृंखला और लिंक के लिए और मार्गदर्शन में वर्तमान वस्तु को हाइलाइट करने के लिए उपयोग किए जाने वाले उच्चारण रंग के साथ किया जाता है।
+
+### रंग थीम एडीटर
+
+Starlight के उच्चारण और ग्रे रंग पैलेट को संशोधित करने के लिए नीचे दिए गए स्लाइडर्स का उपयोग करें।
+गहरे और हल्के पूर्वावलोकन क्षेत्र परिणामी रंग दिखाएंगे, और आपके परिवर्तनों का पूर्वावलोकन करने के लिए पूरा पेज भी अद्यतन हो जाएगा।
+
+जब आप अपने परिवर्तनों से खुश हों, तो नीचे दिए गए CSS या Tailwind कोड को कॉपी करें और इसे अपने परियोजना में उपयोग करें।
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Presets',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
+			default: 'Default',
+			random: 'Random',
+		},
+		editor: {
+			accentColor: 'Accent',
+			grayColor: 'Gray',
+			hue: 'Hue',
+			chroma: 'Chroma',
+			pickColor: 'Pick color',
+		},
+		preview: {
+			darkMode: 'Dark mode',
+			lightMode: 'Light mode',
+			bodyText:
+				'Body text is displayed in a gray shade with a high contrast with the background.',
+			linkText: 'Links are colored.',
+			dimText: 'Some text, like the table of contents, has a lower contrast.',
+			inlineCode: 'Inline code has a distinct background.',
+		},
+	}}
```

Comment:
All the labels in this component can be safely translated (that is actually why they exist so the strings are not hard-coded in the component so it can be easily translated in various languages), e.g. the [Japanese version](https://starlight.astro.build/ja/guides/css-and-tailwind/#%E3%83%86%E3%83%BC%E3%83%9E%E3%82%AB%E3%83%A9%E3%83%BC%E3%82%A8%E3%83%87%E3%82%A3%E3%82%BF) or the French [one](https://starlight.astro.build/fr/guides/css-and-tailwind/#editeur-de-th%C3%A8me-de-couleur).